package.json

```diff
@@ -16,7 +16,8 @@
     "version-packages": "changeset version",
     "publish": "changeset publish",
     "build:studio": "turbo run build --no-cache --filter=studio...",
-    "build:ds": "turbo run build --filter=design-system..."
+    "build:ds": "turbo run build --filter=design-system...",
+    "build:next": "turbo run build --no-cache --filter=studio-next..."
```

Comment:
@Shurtu-gal I suggest `build:studio-next`

Looks like you have installed `next` as a global dependency ?  When I run `npm run build:next` I have this error

```
npm run build:next

> build:next
> turbo run build --no-cache --filter=studio-next...

╭────────────────────────────────────────────────────────────────────────╮
│                                                                        │
│                  Update available v1.10.7 ≫ v1.10.12                   │
│    Changelog: https://github.com/vercel/turbo/releases/tag/v1.10.12    │
│               Run "npx @turbo/codemod update" to update                │
│                                                                        │
│      Follow @turborepo for updates: https://twitter.com/turborepo      │
╰────────────────────────────────────────────────────────────────────────╯
• Packages in scope: studio-next
• Running build in 1 packages
• Remote caching disabled
studio-next:build: cache bypass, force executing baa9d44d992a27f4
studio-next:build:
studio-next:build: > studio-next@0.1.0 build
studio-next:build: > next build
studio-next:build:
studio-next:build: sh: next: command not found
studio-next:build: npm ERR! Lifecycle script `build` failed with error:
studio-next:build: npm ERR! Error: command failed
studio-next:build: npm ERR!   in workspace: studio-next@0.1.0
studio-next:build: npm ERR!   at location: /Users/samir.amzani/Projects/asyncapi/studio/apps/studio-next
studio-next:build: ERROR: command finished with error: command (/Users/samir.amzani/Projects/asyncapi/studio/apps/studio-next) npm run build exited (1)
command (/Users/samir.amzani/Projects/asyncapi/studio/apps/studio-next) npm run build exited (1)

 Tasks:    0 successful, 1 total
Cached:    0 cached, 1 total
  Time:    1.038s
Failed:    studio-next#build

 ERROR  run failed: command  exited (1)
```

Reply:
When I try to install something in the new app 
```
npm install xxx --workspace=react-next

npm ERR! No workspaces found:
npm ERR!   --workspace=react-next

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/samir.amzani/.npm/_logs/2023-08-03T11_54_00_009Z-debug-0.log
➜  studio git:(8a12b9c) npm install nextjs --workspace=react-next
npm ERR! No workspaces found:
npm ERR!   --workspace=react-next

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/samir.amzani/.npm/_logs/2023-08-03T11_54_14_206Z-debug-0.log
```
I saw this happening in an other project, solved by creating a empty workspace: see [this](https://turbo.build/repo/docs/core-concepts/monorepos/code-generation#add-an-empty-workspace)

Reply:
Let me correct it.